libnymea-core/jsonrpc/jsontypes.h

```diff
@@ -106,6 +106,7 @@ class JsonTypes
     enum BasicType {
         Uuid,
         String,
+        StringList,
```

Comment:
The validation of `StringList` seems to be missing in the `validateBasicType()` method.